packages/annotations/src/ui-components/inflection.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div class="alpheios-annotations-comment__container" data-annotation-container>
+    <div class="alpheios-annotations__act-panel">
+      <div class="alpheios-annotations__act-ctrls" :data-annotation-form-open="isFormOpen">
+        <div :data-annotation-selected="isAdding" class="alpheios-annotations__act-ctrls-add"
+             @click.stop="addInflection">[+inflection]</div>
+      </div>
+
+      <div class="alpheios-annotations__act-form" v-show="isAdding">
+        <div class="alpheios-annotations__act-form-content">
+          <div class="alpheios-annotations__act-form-headline">Add an inflection:</div>
+          <div class="alpheios-annotations__act-form-dropdown-group">
+            <label for="latin-number">Number:</label>
+            <select name="latin-number" id="latin-number">
+              <option value="not-selected">Select a value</option>
+              <option value="Singular">Singular</option>
```

Comment:
Just a note for later - - we are going to have to have the domain component tell us what the options are -- they will change depending on the language, particular word, etc.

Reply:
That's a tricky problem we have to solve. The super important questions are where does this knowledge should be and how the annotations would get access to it.